src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/jclouds/JCloudsComputeServiceBuilder.java

```diff
@@ -80,10 +82,16 @@ public ComputeService buildComputeServiceFromInfrastructure(Infrastructure infra
         Iterable<Module> modules = ImmutableSet.of(new SshjSshClientModule());
 
         String domain = infrastructure.getCredentials().getDomain();
-        String identityPrefix = StringUtils.isNotBlank(domain) ? (domain + ":") : "";
-        String identity = identityPrefix + infrastructure.getCredentials().getUsername();
+
+        if (domain != null && !domain.isEmpty()) {
```

Comment:
Please replace this block with the following:

String identityPrefix = StringUtils.isNotBlank(domain) ? (domain + ":") : "";

Reply:
Can you confirm please?